dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -64,14 +64,13 @@ class DdAndroidGradlePlugin : Plugin<Project> {
         variant: ApplicationVariant,
         apiKey: String,
         extension: DdExtension
-    ): Task {
-        val flavorName = if (variant.name.endsWith(SUFFIX_DEBUG)) {
-            variant.name.removeSuffix(SUFFIX_DEBUG)
-        } else if (variant.name.endsWith(SUFFIX_RELEASE)) {
-            variant.name.removeSuffix(SUFFIX_RELEASE)
-        } else {
-            variant.name
+    ): Task? {
```

Comment:
why do you need to return the Task from here ?

Reply:
For the tests :)